src/ocean/core/BitArray.d

```diff
@@ -1063,7 +973,7 @@ struct BitArray
     }
 
     /** ditto */
-    BitArray opCatAssign( BitArray rhs )
+    BitArray opOpAssign(string op)( BitArray rhs ) if (op == "~")
```

Comment:
There is a missed space between the name of the function and the first parenthesis.

Now I see that was missed in initial code, but think, we can fix the style with this PR

Reply:
Also, personally I don't like the style of spaces between parentheses and parameter declaration, but it code style of the `ocean`. So it needs to be respected for template parameters as well.